chunk/chunk_store_test.go

```diff
@@ -56,6 +56,7 @@ func TestChunkStore(t *testing.T) {
 		{"v4 schema", v4Schema},
 		{"v5 schema", v5Schema},
 		{"v6 schema", v6Schema},
+		{"v7 schema", v7Schema},
```

Comment:
Can you add a test to the list of cases below for `{__name__=~".+"}`, `{__name__!="foo"}` and `{}` if valid please.